pkg/tekton/constants.go

```diff
@@ -1,21 +1,25 @@
 package tekton
 
+import (
+	v1 "github.com/jenkins-x/jx/pkg/apis/jenkins.io/v1"
+)
+
 const (
 	// LastBuildNumberAnnotationPrefix used to annotate SourceRepository with the latest build number for a branch
 	LastBuildNumberAnnotationPrefix = "jenkins.io/last-build-number-for-"
 
+	// LabelOwner is the label added to Tekton CRDs for the owner of the repository being built.
+	LabelOwner = v1.LabelOwner
+
+	// LabelRepo is the label added to Tekton CRDs for the repository being built.
+	LabelRepo = v1.LabelRepository
```

Comment:
@abayer this is the gist of the change. I kept the tekton constant file since some things we still need. For the constants which apply for PipelineActivity as well,  I am referencing the issues under "github.com/jenkins-x/jx/pkg/apis/jenkins.io/v1"

Reply:
Using  the golden file approach worked like a charm :-)